lib/yabeda/active_record.rb

```diff
@@ -71,7 +71,12 @@ class Error < StandardError; end
 
       # Connection pool metrics collection
       collect do
-        connection_pools = ::ActiveRecord::Base.connection_handler.connection_pool_list
+        connection_pools = \
+          if Rails.version >= "7.1"
+            ::ActiveRecord::Base.connection_handler.connection_pool_list(:all)
```

Comment:
I don't know much about this code but I assume we want to retrieve all roles here, if instead we want to only use the current role we can achieve it via `::ActiveRecord::Base.connection_handler.connection_pool_list(ActiveRecord::Base.current_role)`